lib/framework/stub/contacts_manager.dart

```diff
@@ -20,8 +20,7 @@ class ContactManagerStub extends ContactManager {
 
   @override
   Future<bool> requestPermission() {
-    throw ConfigError(
-        "Phone Contact Service is not enabled. Please review the Ensemble documentation.");
+    return Future.value(false);
```

Comment:
is this checking for permission or request user access? If it's the latter you'd want to throw the Error right?

Reply:
According to the doc, there's no dedicated method to check for the permission is enabled or not in the flutter_contacts package.

FYI, See the following
1. https://pub.dev/documentation/flutter_contacts/latest/flutter_contacts/FlutterContacts-class.html
2.https://github.com/QuisApp/flutter_contacts/blob/b1fd688c34968c30e76977479e4ed392bcb3bfda/ios/Classes/SwiftFlutterContactsPlugin.swift#L447C15-L447C32

There's a way to do that natively, But this package didn't expose the method `authorizationStatus(for:)` to check the permission which returns `CNAuthorizationStatus` - https://developer.apple.com/documentation/contacts/cncontactstore/1403143-authorizationstatus